canonical/canonical.com
# [WD-16470] Page refresh microcloud

## Done

- Copy update
- Changed Youtube Link
- Added a new quote section
- Update logo assets

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/microcloud
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Make sure that the page matches the design: https://www.figma.com/design/hzQYJM9Tui4wpmyZC0k0WS/23.10-C.com---MicroCloud-Rebranding?node-id=0-1&node-type=canvas&t=gTy0rbPmisXRTtBj-0
- Make sure that the changes in this copy doc are applied: https://docs.google.com/document/d/17iBbFCINLYjPBOYWpgoq9SreyYNkR0Sb5jgx5iTo6MA/edit?tab=t.0


## Issue / Card

Fixes WD-16470
